bitnami/minio/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: minio
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/minio
   - https://min.io
-version: 11.10.25
+version: 11.10.26
```

Comment:
Hi @cedi,

Version `11.10.26` has been already released: https://github.com/bitnami/charts/blob/main/bitnami/minio/Chart.yaml

Could you please bump the chart version again?